public/app/features/alerting/unified/hooks/useExternalAmSelector.ts

```diff
@@ -4,7 +4,7 @@ import { StoreState } from '../../../../types';
 const SUFFIX_REGEX = /\/api\/v[1|2]\/alerts/i;
 type AlertmanagerConfig = { url: string; status: string; actualUrl: string };
 
-export function useExternalAmSelector(): AlertmanagerConfig[] | undefined {
+export function useExternalAmSelector(): AlertmanagerConfig[] | [] {
```

Comment:
Not strictly necessary as `AlertmanagerConfig[]` also allows the array to be empty :)